docs/run-platform-server/monitoring.mdx

```diff
@@ -7,11 +7,54 @@ import { CodeExample } from "@site/src/components/CodeExample";
 
 ## Metrics
 
-Metrics are emitted from a running Horizon process and exposed _privately_ via the `/metrics` path, accessible only through the Horizon admin port bound to host machine. You need to add environment configuration parameter `ADMIN_PORT=XXXXX` , since it's disabled by default, then metrics are published on the host machine as `localhost:<ADMIN_PORT>\metrics`.
+Metrics are emitted from Horizon over HTTP in [the de facto text based exposition format](https://github.com/prometheus/docs/blob/main/content/docs/instrumenting/exposition_formats.md#text-based-format). The Metrics are published on the _private_ `/metrics` path of Horizon Admin port, which is an optional service to be started and will be bound by the horizon process onto the host machine loopback network(localhost or 127.0.0.1). To enable the Admin port, add environment configuration parameter `ADMIN_PORT=XXXXX`, the metrics endpoint will be reachable on the host machine as `localhost:<ADMIN_PORT>/metrics`. You can verify this by pointing any browser that can reach this address, it will print out all metrics keys.
+
+### Exporting
+
+Once the admin port is enabled, the Horizon metrics endpoint can be 'scraped' by external monitoring infrastructure. Since the metrics output is encoded to [standard text based format](https://github.com/prometheus/docs/blob/main/content/docs/instrumenting/exposition_formats.md#text-based-format) it will be compatibile for usage with many types of monitoring infrastructure that interoperate with same standard format.
```

Comment:
```suggestion
Once the admin port is enabled, the Horizon metrics endpoint can be 'scraped' by external monitoring infrastructure. Since the metrics output is encoded to [standard text based format](https://github.com/prometheus/docs/blob/main/content/docs/instrumenting/exposition_formats.md#text-based-format) it will be compatible for usage with many types of monitoring infrastructure that interoperate with the same standard format.
```